### PR TITLE
time, date, mkdir, print: add Russian translation

### PR DESCRIPTION
**Pull request by AStmav**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution. The pages LGTM, if possible can you translation the placeholders inside `{{}}` too?

**Comment by AStmav** [comment]
> Welcome to tldr and thanks for your contribution. The pages LGTM, if possible can you translation the placeholders inside `{{}}` too?

Yes of course I made changes

File: pages.ru/windows/print.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+> Вывод на печать текстового файла.
+> Больше информации: <https://learn.microsoft.com/windows-server/administration/windows-commands/print>.
+
+- Печать текстового файла используя локальный принтер:
+
+`print {{путь\до\каталога}}`
+
+- Печать текстового файла используя сетевой принтер:
+
+`print /d:{{принтер}} {{путь\до\каталога}}`
```
Comment (kbdharun) [inline pages.ru/windows/print.md:12]:
```suggestion
`print /d:{{принтер}} {{путь\до\папки}}`
```

File: pages.ru/windows/print.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# print
+
+> Вывод на печать текстового файла.
+> Больше информации: <https://learn.microsoft.com/windows-server/administration/windows-commands/print>.
+
+- Печать текстового файла используя локальный принтер:
+
+`print {{путь\до\каталога}}`
```
Comment (kbdharun) [inline pages.ru/windows/print.md:8]:
```suggestion
`print {{путь\до\папки}}`
```

File: pages.ru/windows/mkdir.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+> Создает каталог или подкаталог в файловой системе.
+> Больше информации: <https://learn.microsoft.com/windows-server/administration/windows-commands/mkdir>.
+
+- Создать новый каталог:
+
+`mkdir {{путь\до\каталога}}`
+
+- Чтобы создать дерево каталогов в корневом каталоге введите:
+
+`mkdir {{путь\до\под_каталога}}`
```
Comment (kbdharun) [inline pages.ru/windows/mkdir.md:12]:
```suggestion
`mkdir {{путь\до\папки}}`

- Чтобы создать дерево каталогов в корневом каталоге введите:

`mkdir {{путь\до\под_папки}}`
```

**Review by kbdharun — COMMENTED** [review]
**Ref**. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

**Review by gutjuri — APPROVED** [review]
LGTM after the suggestions!

**Comment by kbdharun** [comment]
@AStmav Any updates on this?

**Comment by AStmav** [comment]
Confirmed suggested change. Sorry for the long answer

> Hi @AStmav, Any updates on this?

Confirmed suggested change. Sorry for the long answer

**Review by kbdharun — APPROVED** [review]

**Pull request merged**